generatorv2/lib/generator.rb

```diff
@@ -0,0 +1,49 @@
+require 'toml-rb'
```

Comment:
What problems or solutions are we avoiding/gaining by using TOML as opposed to YAML (or even JSON)?  This is the point at which TOML as a tool and dependency is being introduced, and its associated maintenance cost, so we should evaluate that.  We should have a net positive effect by bringing this in.

Reply:
The `test.toml` files are written in toml, using another file format isn't really an option since rewriting it would require parsing it. 

I wrote a manual parser for the Crystal generator (but it is likely not as good of an implementation) because I couldn't find any good shards (library). This library should be fine, the toml specification isn't changing, but we could have our own solution.

Reply:
The goal is more or less with the toml files is that, if someone adds extra test cases or reimplements test cases for a certain exercise will that not affect the generation process until you sync the exercise. Meaning the ci won't fail in the majority of cases if not all exercises are synced.